src/Plugin/Condition/DataComparison.php

```diff
@@ -21,14 +21,14 @@
  *       label = @Translation("Data to compare"),
  *       description = @Translation("The data to be checked to be empty, specified by using a data selector, e.g. 'node:uid:entity:name:value'.")
  *     ),
+ *     "value" = @ContextDefinition("any",
+ *        label = @Translation("Data value"),
+ *        description = @Translation("The value to compare the data with.")
+ *     ),
  *     "operator" = @ContextDefinition("string",
  *       label = @Translation("Operator"),
  *       description = @Translation("The comparison operator."),
  *       required = FALSE
```

Comment:
We must keep the operator second, as we'll need it that way in future (operator plugins that influence what you can select after). Let's better make it required instead - it should be really, but have a default value! NULL is not a valid value for it. Might be we have to adapt tests also for that.